tree/ntuple/v7/test/CMakeLists.txt

```diff
@@ -22,6 +22,7 @@ if(MSVC)
                                      ${CMAKE_CURRENT_BINARY_DIR}/libCustomStruct.dll)
 endif()
 ROOT_ADD_GTEST(ntuple ntuple.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore CustomStruct)
+ROOT_ADD_GTEST(ntuple_compression ntuple_compression.cxx LIBRARIES ROOTDataFrame ROOTNTuple MathCore)
```

Comment:
If `TRandom` is not used, the `MathCore` dependency can be removed. The data frame dependency seem to be avoidable, too.